ci_framework/roles/dlrn_report/tasks/main.yml

```diff
@@ -14,8 +14,11 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-- name: Install dlrnapi-client
-  ansible.builtin.import_tasks: install.yml
+- name: Only report DLRN results when var is set
+  when: cifmw_dlrn_report_result | bool
```

Comment:
FTR, this is a default value: https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci_framework/roles/dlrn_report/README.md?plain=1#L19

https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci_framework/roles/dlrn_report/defaults/main.yml#L24

Reply:
Agree, should be nice to have it written there

Reply:
That's ok - the base jobs set to false